### PR TITLE
Use _url instead of _path

### PR DESCRIPTION
**Pull request by mtsmfm**
context: https://github.com/bm-sms/daimon-news-multi-tenant/pull/206#discussion_r51962308

`git grep --name-only ' [a-z_]*_path' | grep -v test | xargs -L1 ruby -i -e 'puts gets(nil).to_s.gsub(/( [a-z_]*)_path(?!:)/, "\\1_url")'`

### CODE REVIEW

**Comment by tricknotes** [comment]
Seems good :sunny:

File: app/views/posts/show.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - if @pages.respond_to?(:total_pages)
   - content_for(:rel_next_prev_link_tags) { rel_next_prev_link_tags(@pages) }
   - content_for(:canonical_tag) do
-    %link{rel: :canonical, href: url_for(all: true, only_path: false)}
+    %link{rel: :canonical, href: url_for(all: true, only_url: false)}
```
Comment (okkez) [inline app/views/posts/show.html.haml]: :eyes: Is this correct?

Comment (mtsmfm) [inline app/views/posts/show.html.haml]: OMG

Comment (mtsmfm) [inline app/views/posts/show.html.haml]: I'll add test for `only_path` option

Comment (okkez) [inline app/views/posts/show.html.haml]: http://api.rubyonrails.org/classes/ActionDispatch/Routing/UrlFor.html#method-i-url_for

`only_path: false` is correct for this purpose.

**Comment by mtsmfm** [comment]
Added test

**Comment by okkez** [comment]
LGTM.

**Pull request merged**